osgp/platform/osgp-throttling-service/src/test/java/org/opensmartgridplatform/throttling/MaxConcurrencyByBtsCellConfigTest.java

```diff
@@ -0,0 +1,64 @@
+// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.opensmartgridplatform.throttling;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensmartgridplatform.throttling.entities.BtsCellConfig;
+import org.opensmartgridplatform.throttling.repositories.BtsCellConfigRepository;
+
+@ExtendWith(MockitoExtension.class)
+class MaxConcurrencyByBtsCellConfigTest {
+  @InjectMocks private MaxConcurrencyByBtsCellConfig btsCellConfig;
+  @Mock private BtsCellConfigRepository btsCellConfigRepository;
+
+  @Test
+  void shouldFetchFromCache() {
+    final int btsId = 5;
+    final int cellId = 6;
+    final int maxConcurrency = 9;
+    this.btsCellConfig.setMaxConcurrency(btsId, cellId, maxConcurrency);
+
+    assertThat(this.btsCellConfig.getMaxConcurrency(btsId, cellId))
+        .isEqualTo(Optional.of(maxConcurrency));
```

Comment:
Should verify that this.btsCellConfigRepository.findByBaseTransceiverStationIdAndCellId is never called.

Reply:
Added: verifyNoInteractions(this.btsCellConfigRepository);